### PR TITLE
Issue #283: Fix notice errors, regression introduced in #269.

### LINKED ISSUE #283: [regression] Fix notice error on atomium-overview page
Introduced in #269, there are notice errors related to the pager component.

### PR DESCRIPTION
**Pull request by drupol**
Fix #283

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -1,3 +1,13 @@
+# ==============================================================================
```
Comment (aritomelo) [inline .drone.yml]: Why these changes belong to this issue?

Comment (aritomelo) [reply .drone.yml]: Also, there is a ticket for drone cache issue which will be a good candidate for this issue.
[Jira Ticket](https://webgate.ec.europa.eu/CITnet/jira/browse/NEPT-2288)

Let me know your thought @drupol.

Comment (drupol) [reply .drone.yml]: That particular issue was for the package `drupal-composer/preserve-paths` I guess it can be closed now.

Comment (drupol) [reply .drone.yml]: These changes were pushed into this issue to see if the build would be successfull.
I will create a new PR with those changes only now.

Comment (drupol) [reply .drone.yml]: fixed in #292
